numismatic/feeds/base.py

```diff
@@ -72,7 +72,7 @@ def _rest_client_validator(self, attribute, value):
     @websocket_client.validator
     def _websocket_client_validator(self, attribute, value):
         self.websocket_client = None if self._websocket_client_class is None \
-            else self._websocket_client_class(websocket=self.websocket)
+            else self._websocket_client_class()
```

Comment:
These vallidators seem very "hacky" to me. I would like to remove them

Reply:
It's one method that `attrs` provides of initialising an attribute based on other attributes. I agree that the term *validator* isn't right but if you look beyond that then I think it is fine.